samples/artifacts/DataSummarization/ExternalTimeBatchWindowStdDev.siddhi

```diff
@@ -0,0 +1,46 @@
+@App:name("ExternalTimeBatchWindowStdDev")
+@APP:description("Demonstrates how to calculate the standard deviation of events by using a Siddhi external time batch window.")
```

Comment:
Change it to "Calculate the standard deviation based on event timestamps"

Reply:
6abc5699d9a9c3ddf8f5e200716a4e7c236ef463 This commit fixed the above corrections